src/panels/profile/ha-refresh-tokens-card.ts

```diff
@@ -43,7 +55,7 @@ class HaRefreshTokens extends LitElement {
       <div class="card-content">
         ${this.hass.localize("ui.panel.profile.refresh_tokens.description")}
         ${refreshTokens?.length
-          ? refreshTokens!.map(
+          ? refreshTokens!.sort(compareTokenLastUsedAt).map(
```

Comment:
We should do the sorting in the `memoizeOne`, to prevent doing it on every render, and only do it when the tokens are changed

Reply:
That makes sense. So change:

```ts
refreshTokens?.filter((token) => token.type === "normal").reverse()
```

to 

```ts
refreshTokens?.filter((token) => token.type === "normal").sort(compareTokenLastUsedAt)
```

?

Reply:
```ts
private _sortedTokens = memoizeOne((tokens) => tokens.sort(compareTokenLastUsedAt).map())
```

Something like this. Break it into a private function

Reply:
there is a `memoize` function already :-)

Reply:
@aav7fl yes, thats perfect!

Reply:
@bramkragten Change pushed. Thanks for walking through with me on this.